### PR TITLE
Ml quickstart

### PR DESCRIPTION
**Pull request by leemc-data-ed**
Added a quick start guide! Also some minor tweaks to the landing page of the main Orientation doc.   Link to check it out *in situ*: 

* Main Orientation doc:  https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/ml-quickstart/arcus_orientation.md#1. 
* Quickstart: https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/ml-quickstart/arcus_quickstart.md#1

### CODE REVIEW

File: arcus_orientation.md [hunk 1]
```diff
@@ -25,33 +25,23 @@ script: https://kit.fontawesome.com/83b2343bd4.js
...
-
-**Estimated time to completion**: @estimated_time
-
-**Pre-requisites**: It's helpful to have reviewed the Arcus website at https://arcus.chop.edu (available only on the CHOP network), to understand Arcus overall goals.
-
-**Learning Objectives**:
+### Quick Start Guide
 
-@learning_objectives
+Just want to jump into how to get an Arcus Lab, or using one you already have? Go to [our quick start guide](arcus_quickstart.md).
```
Comment (pm0kjp) [inline arcus_orientation.md:38]:
```suggestion
Just want to jump into how to get an Arcus Lab, or using one you already have? Go to [our quick start guide](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/main/arcus_quickstart.md#1). 
```

Note that currently the link goes to the raw markdown file and you want it to go to the liascript.  The link I put in won't work YET because it goes to main, but just swap "main" for "ml-quickstart" in the URL to see current state.

File: arcus_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+- Save time: Hone your data request beforehand
+- Know the rules: Security, privacy, and the [Arcus Terms of Use](https://arcus.chop.edu/terms-of-use).
+
+### Is your proposed study a good fit for Arcus?
+
+<div style = "background-color:white;">
+
+<script style="display: block" run-once="true" modify="false">
+mermaid.initialize({});
+
```
Comment (pm0kjp) [inline arcus_quickstart.md:39]: A few notes on the diagram:

* Mention NTP status in the "chop login credentials" e.g. "everyone has (or everyone can get, through NTP status) chop login"
* Have a descriptive no for each no, if possible, like, "no, there are some critical collaborators that I can't get NTP status for", "no, my project is operational or quality improvement, or otherwise not research", "no, we need a special tool that isn't supported by Arcus", etc.
* Make the tools question broader, like "we're able to use the Arcus toolkit, or we're willing to hire someone who can,"
* The final "not a good fit" should be softened: "Arcus might not be the right fit, depending on your circumstances." or similar...
* In short, we're aiming for helpful but not discouraging.
Also, non-human research is going to be supported in some degree in the future so stay tuned, I have an ask in to Marianne about this.

File: arcus_quickstart.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+When you [request an Arcus Lab](https://support.arcus.chop.edu/servicedesk/customer/portal/6/create/307), we evaluate your proposed study to make sure it's a good fit for our service, then build the computational environment and provision the data for your team so you can start work. 
+This process takes time and coordination across multiple groups within Arcus. 
+Here's what you can do to make the process as smooth as possible: 
+
+- Check first: Is your proposed study a good fit for Arcus?
+- Save time: Hone your data request beforehand
+- Know the rules: Security, privacy, and the [Arcus Terms of Use](https://arcus.chop.edu/terms-of-use).
+
+### Is your proposed study a good fit for Arcus?
```
Comment (pm0kjp) [inline arcus_quickstart.md:33]: Feedback from managers' meeting: swap it, so it doesn't feel like we're pointing fingers at them being not good enough, e.g. "Is Arcus a good fit for your study".

File: arcus_quickstart.md (hunk 2 above)
Comment (pm0kjp) [reply arcus_quickstart.md:39]: UPDATE.  Remove entirely the "human data" question, we will and can accomodate animal models or wet lab projects.

OR, replace it with a question (we went back and forth on how to phrase this in the managers' meeting) that is something along the lines of "Can you describe your data clearly enough so that we can gather it for you"

**Comment by rosemm** [comment]
A couple quick options for making the landing page more visually attractive: 

- [option 1](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/e60aebd4686814ade38835ae431582fa60583637/arcus_quickstart.md#2)
- [option 2](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/5c06e08d9a1283c75a0b0870383867f6217e1991/arcus_quickstart.md#2)
- [option 3](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/49142ac984ed1c531f9ed9e8b6d50e95879beeb3/arcus_quickstart.md#2)

File: arcus_quickstart.md (hunk 2 above)
Comment (pm0kjp) [reply arcus_quickstart.md:39]: Another update / feedback from managers' meeting: Maybe change the tools question to "Are you prepared to clean and analyze data using Arcus tools" bc the tools are related to cleaning and analyzing data, not, for example, for data collection.  

Managers also want NTP info added to the first question (even though you talk about NTP later, just to make sure it doesn't turn people off too quickly.

File: arcus_quickstart.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+link:  https://cdn.jsdelivr.net/gh/arcus/Arcus_Labs_Orientation@main/assets/styles.css
+link:  https://cdn.jsdelivr.net/gh/arcus/education_modules@main/assets/styles.css
+script: https://kit.fontawesome.com/83b2343bd4.js
+script: https://cdn.jsdelivr.net/npm/mermaid@9.4.3/dist/mermaid.min.js
+-->
+
+# Arcus Labs: Quick Start
+
+Ready to start a research project with Arcus, or have you been given a computational lab and want to know what to do next? This quick start guide will jump start your Arcus journey!
```
Comment (pm0kjp) [inline arcus_quickstart.md:17]: Maybe add a description of "what is Arcus" at the very top.  In managers' meeting we joked that no one has been successful at this so honestly no pressure to get it right.  But perhaps something like "Arcus labs are a place where we can aggregate data from various sources and place that data alongside useful analysis tools.  The goal is to set you and your team up with an environment that supports reproducible data analysis projects and simplifies sharing your research insights, cohorts, data, and methods.  This can in turn help your scientific reputation and help you get the most out of your data."  IDK.

File: arcus_quickstart.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+"HTML: " + svg
+</script>
+
+</div>
+
+Let's go into a bit more detail about each of these questions. 
+
+#### Does everyone on your team have CHOP login credentials?
+Anyone who will need access to the data has to have CHOP credentials. 
+You will not be able to share the data with outside collaborators unless they first get NTP status with CHOP.
```
Comment (pm0kjp) [inline arcus_quickstart.md:67]: Maybe link to where PIs can learn more about NTP requests?  Also mention (I think this is true) that you have to specify that NTPs get email addresses / CHOP logins.  I suspect there's a good Forum post about this (whose links probably need checking at this point)

**Review by pm0kjp — APPROVED** [review]
I reviewed the changes and really like them, especially the mermaid chart!  Looks good to me.
